.github/workflows/cpu_tests.yml

```diff
@@ -122,6 +122,19 @@ jobs:
       - name: "Filter tests"
         if: ${{ needs.pre_job.outputs.should_skip == 'false' }}
         uses: ./.github/actions/test_filter
+      - name: "Set CMAKE_PREFIX_PATH"
+        if: ${{ needs.pre_job.outputs.should_skip == 'false' }}
+        run: |
+          if [[ "${{ matrix.toolchain}}" == *"openmp"* ]]; then
+            dirs=$(find /opt/openmp -name '*.cmake' | xargs dirname | sort -u | paste -sd ':' -)
```

Comment:
Looking at https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure, it seems to me CMake officially searches directly in `<prefix>` only on Windows.

What about explicitl listing all libraries ?

Reply:
> Looking at https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure, it seems to me CMake officially searches directly in <prefix> only on Windows.

While testing I used `find_package(X REQUIRED)` so errors were raised if the package was not found. I am therefore certain that this is working as expected. Plus we can see in the tests that kokkos-kernels is not recompiled

> What about explicitl listing all libraries ?

Is there a reason to do this? Doesn't that just give us an extra list to maintain?

Reply:
> Is there a reason to do this?

Well, we are out of the documented behavior.

Reply:
Beacause it only officially searches directly in `<prefix>` on Windows?

How does explicitly listing all libraries fix this? The result in CMAKE_PREFIX_PATH would be the same wouldn't it?

Reply:
> Beacause it only officially searches directly in `<prefix>` on Windows?

Yes, the typical usage is to match the prefix path used during the installation stage.

> How does explicitly listing all libraries fix this? The result in CMAKE_PREFIX_PATH would be the same wouldn't it?

My idea was to list all root prefixes manually. If all paths exhibit the same structure, maybe we can simply get the root directory of each package by appending some `../` ?

Reply:
`CMAKE_PREFIX_PATH=/opt/openmp` works once the paths are modified